experimental/dds/tree2/src/feature-libraries/editable-tree-2/lazyTree.ts

```diff
@@ -509,6 +509,16 @@ const cachedStructClasses = new WeakMap<
 	) => LazyStruct<StructSchema>
 >();
 
+export function getBoxedField(
+	struct: LazyEntity,
+	key: FieldKey,
+	fieldSchema: FieldSchema,
+): TreeField {
+	return inCursorField(struct[cursorSymbol], key, (cursor) => {
+		return makeField(struct.context, fieldSchema, cursor);
+	});
+}
```

Comment:
@CraigMacomber - I lifted this function as a "back door" to force an optional field into existence in order to set it.  LMK if you have a better alternative.